src/LookupTable/hooks/useGetSingleTableStructure.hook.tsx

```diff
@@ -28,7 +29,11 @@ const useGetTableStructure = (): LookupTableStructureType => {
 
   useEffect(() => {
     if (!loading && !error && data?.lookupTable) {
-      setStructure(data.lookupTable as any)
+      const lookupTable = data.lookupTable
+      const namePlural = pluralize.isPlural(lookupTable.name)
+        ? lookupTable.name
+        : pluralize.plural(lookupTable.name)
+      setStructure({ ...lookupTable, name: namePlural })
```

Comment:
I think the reason why table names were converted to plural is because the GraphQL schema has a pluralised node for returning multiple entities of a type (as in `application` for a single one and `applications`). However, forcing the table name to be plural actually *restricted* the GraphQL schema, cos you couldn't easily differentiate between a single item and a list of items, and being able to search by a single item is probably more likely within application forms.

So this addition here just converts the table name to plural form for querying the "multiple" node for that entity (e.g `products`)